### PR TITLE
Update data-classification-content-explorer.md

### PR DESCRIPTION
**Pull request by shparshw**
Updating RBAC documentation for content explorer roles.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/data-classification-content-explorer.md [hunk 1]
```diff
@@ -42,13 +42,14 @@ A [retention label](labels.md) allows you to define how long a labeled item is k
 
 ### Permissions
 
-There are two roles that grant access to Content explorer:
+There are two role roups that grant access to Content explorer:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-content-explorer.md]:
```suggestion
There are two role groups that grant access to Content Explorer:
```

File: microsoft-365/compliance/data-classification-content-explorer.md [hunk 2]
```diff
@@ -42,13 +42,14 @@ A [retention label](labels.md) allows you to define how long a labeled item is k
...
 
-- **Content Explorer Content viewer**: Membership in this role allows you to view the contents of each item in the list.
+- **Content Explorer Content viewer**: Membership in this role group allows you to view the contents of each item in the list.
 
-The account you use to access Content explorer must be in one or both of the roles. These are independent roles and are not cumulative. For example, if you want to grant an account the ability to view the items and their locations only, grant Content Explorer List viewer rights. If you want that same account to also be able to view the contents of the items in the list, grant Content Explorer Content viewer rights as well.
+The account you use to access Content explorer must be in one or both of the role groups. These are independent role groups and are not cumulative. For example, if you want to grant an account the ability to view the items and their locations only, grant Content Explorer List viewer rights. If you want that same account to also be able to view the contents of the items in the list, grant Content Explorer Content viewer rights as well.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-content-explorer.md]:
```suggestion
The account you use to access Content Explorer must be in one or both of the role groups. These are independent role groups and are not cumulative. For example, if you want to grant an account the ability to view the items and their locations only, grant Content Explorer List viewer rights. If you want that same account to also be able to view the contents of the items in the list, grant Content Explorer Content viewer rights as well.
```

File: microsoft-365/compliance/data-classification-content-explorer.md [hunk 3]
```diff
@@ -42,13 +42,14 @@ A [retention label](labels.md) allows you to define how long a labeled item is k
...
+Apart from role groups, **Data Classification List viewer** and **Data Classification Content viewer** are user roles which can be added to new custom role group or any existing role group to manage access to content explorer.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-content-explorer.md]:
```suggestion
Apart from role groups, **Data Classification List viewer** and **Data Classification Content viewer** are user roles which can be added to a new custom role group or any existing role group to manage access to Content Explorer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by chrfox** [comment]
typos fixed in separate PR

**Pull request closed without merging**